packages/sdks/web-component/src/app/index.html

```diff
@@ -96,10 +98,10 @@
         //   const formData = new FormData(e.target);
         //   const data = Object.fromEntries(formData.entries());
 
-        //   next('Ppb_65tyyn', data);
-        // });
+        //   next('pXVwWREG7M', data);
+        // }, { once: true });
```

Comment:
in this example - is the `once: true` is by purpose?

this means that the developer will have to add it every time

Reply:
Yes, it's on purpose 
Otherwise, every time this fn is called we will add another listener to the submit